### ISSUE TITLE
Suggestion: Switch to Jupyter Labs (No supporting changes needed)

### ISSUE THREAD

**Issue by codefaux** [borisdayma/dalle-mini#277]
Dunno if anyone else noticed but Jupyter has a new thing -- Jupyter Labs. It's a much better interface overall. Zero compatibility issues. Much nicer -- seriously, much much more friendly to use. Closer to Google Colab.

HAS A DARK THEME.
Has large output view panels by default.

Dockerfile:
- RUN pip install jupyter
+RUN pip install jupyterlabs
+RUN pip install ipywidgets

run_infer_notebook.sh:
- jupyter notebook --ip 0.0.0.0 --no-browser --allow-root
+ jupyter labs --ip 0.0.0.0 --no-browser --allow-root

Zero other changes required to the image, notebook, or model, and it is immediately SO MUCH BETTER to use.

**Comment by borisdayma** [comment]
Yeah that's why I have the raw notebook in the repo. In fact the colab just pulls the file from this repo.
I personally really like VS Code.
